functions/go/export-terraform/terraformgenerator/templates/log-export.tf

```diff
@@ -45,4 +45,14 @@ module "{{ .GetResourceName }}-destination" {
     is_locked             = {{ .GetBool "spec" "retentionPolicy" "isLocked" }}
   }{{end}}
 }
+{{end}}{{with $logsink.References.LoggingLogBucket }}
+module "{{ .GetResourceName }}-destination" {
+  source  = "terraform-google-modules/log-export/google//modules/logbucket"
+  version = "~> 7.3.0"
+
+  project_id                  = module.{{ .Parent.GetResourceName }}.project_id
+  log_bucket_name             = "{{ .GetResourceName }}"
+  location                    = "{{.}}"{{end}}{{ with .GetStringFromObject "spec" "location" }}
+  retention_days              = "{{.}}"{{end}}{{ if .GetInt "spec" "retentionDays"}}
+}
```

Comment:
We should also expose https://github.com/GoogleCloudPlatform/blueprints/blob/aa4a37d846c816de4649fa555c327f9ede89cb26/catalog/log-export/folder/logbucket-export/export.yaml#L58 in TF for parity above with the storage

Reply:
I wanted to but it's not exposed in the [TF resource](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/logging_project_bucket_config) for some reason. I'll open up an issue in the provider repo but for now we'll have to let this go.

Reply:
@g-awmalik sgtm

Reply:
https://github.com/hashicorp/terraform-provider-google/issues/11798